src/openzaak/components/catalogi/migrations/0007_auto_20191119_1136.py

```diff
@@ -0,0 +1,43 @@
+# Generated by Django 2.2.4 on 2019-11-19 11:36
+
+from django.db import migrations
+from django.db import models
+
+
+class Migration(migrations.Migration):
```

Comment:
Can you please rebase on master and update/merge migrations? Otherwise after the merge migrations would fail